app/models/accounting/qb/transaction_reconciler.rb

```diff
@@ -35,7 +35,7 @@ def builder
       end
 
       def service(m_txn)
-        m_txn.subtype?("Check") ? purchase_service : je_service
+        m_txn.qb_object_type == "Purchase" ? purchase_service : je_service
```

Comment:
It is unlikely but possible there will be legacy disbursements that are JEs, so we decide based on qb object type, not txn type